magellan-compose/src/main/java/com/example/magellan/compose/ComposeStep.kt

```diff
@@ -0,0 +1,21 @@
+package com.example.magellan.compose
```

Comment:
nit: the naming of the classes is a bit confusing. But I can't think of better names. Something we might want to revisit later.